Harvest.Core/Domain/Project.cs

```diff
@@ -114,13 +122,13 @@ internal static void OnModelCreating(ModelBuilder modelBuilder)
                 .WithMany(p => p.Quotes)
                 .HasForeignKey(q => q.ProjectId)
                 .OnDelete(DeleteBehavior.Restrict);
-
         }
         public class Statuses
         {
             public const string Requested = "Requested";
             public const string PendingAccountApproval = "Pending Account Approval";
-            public const string Active    = "Active";
+            public const string Active = "Active";
+            public const string ChangeRequested = "ChangeRequested";
```

Comment:
Add a space in the string?
Also, add it to the TypeList? (Not a big deal as I don't think we are using that yet if ever.)

Reply:
I don't like spaces in the string, makes the db match harder too.  but I should probably change the others so it's consistent